docs/versioned-plugins/filters/unique-v2.0.5.asciidoc

```diff
@@ -20,6 +20,7 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
+The unique filter.
```

Comment:
Description is missing here. Need to add a *real* description for this plugin...I've just added a placeholder.